phoonnx_train/optispeech/utils/generic.py

```
@@ -0,0 +1,258 @@
+            }
+        )
+        start = duration
+
+    assert list(duration_json[-1].values())[0]["endtime"] == sum(
+        durations
+    ), f"{list(duration_json[-1].values())[0]['endtime'], sum(durations)}"
+    return duration_json
+
+
+def get_model_size_mb(model):
+    buf = io.BytesIO()
```

Ruff (0.15.1)

[error] Undefined name `io`

(F821)